.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
[OPTIONAL FOR THIS PROJECT ONLY]

- [ ] From now on, in all projects we build, we need to have set in the `.gitignore` file the `node_modules/` folder. It is recommended to add it here too.